tests/functional/common.config

```diff
@@ -72,6 +72,11 @@ export SED_PROG="`set_prog_path sed`"
 export BC_PROG="`set_prog_path bc`"
 [ "$BC_PROG" = "" ] && _fatal "bc not found"
 
+PGREP_OPTIONS='-a'
+if pgrep * -a 2>&1 | grep -q "invalid option";then
```

Comment:
This commit breaks tests/fucntional on my environment (pgrep 3.3.9). * seems to be an invalid parameter for the command. How about "-u root" for checking "-a" is valid or not?